Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jenkins:2.60.2
+FROM jenkins/jenkins:lts
```

Comment:
This should be a specific version, such as `jenkins/jenkins:2.60.3`. This allows repeatable builds and ensures CI is building and packaging the intended Jenkins version and plugins.